### ISSUE TITLE
LD Reference parameter use is mixed up

### ISSUE THREAD

**Issue by CristiGuijarro** [corbinq/GAMBIT#1]
## The Issue

Whilst trying to run `GAMBIT` on other (non default) LD reference panels for other populations I noticed that the software kept defaulting to ["G1K_EUR_3V5/chr$.vcf.gz"](https://github.com/corbinq/GAMBIT/blob/master/src/Main.cpp#L45). This was because I was trying to pass in the `--ldref` parameter rather than `--ldfile`. This incorrect usage is also referenced here: https://xqwen.github.io/ptwas/ - the "Quick Start" use case I was following.

## Lines of interest

https://github.com/corbinq/GAMBIT/blob/1644b80d252d9def1ae2fdf85067d53d06162966/src/Main.cpp#L18

- https://github.com/corbinq/GAMBIT/blob/master/src/Main.cpp#L98

- https://github.com/corbinq/GAMBIT/blob/master/src/Main.cpp#L191

## TLDR;
- in the line above:
Should be `--ldfile` rather than `--ldref`
